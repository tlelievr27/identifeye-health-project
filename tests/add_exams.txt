ADD PATIENT 123 JON DOE
ADD PATIENT 124 JANE DOE
ADD PATIENT 555 MARY SUE
ADD EXAM 123 555
ADD EXAM 124 556
ADD EXAM 123 557
ADD EXAM 555 604
ADD PATIENT 123 FAKE JON DOE
ADD EXAM 124 556
ADD EXAM 125 600
ADD EXAM 123 556