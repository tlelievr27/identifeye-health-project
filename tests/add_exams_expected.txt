Name: JOHN DOE, Id: 123, Exam Count: 2
Name: JANE DOE, Id: 124, Exam Count: 1
Name: MARY SUE, Id: 555, Exam Count: 1